docs/UsersGuide.asciidoc

```diff
@@ -1031,6 +1031,23 @@ openqa-dump-templates --json --group test > test.json
 openqa-load-templates test.json
 --------------------------------------------------------------------------------
 
+== Timers and triggers ==
+
+Cleanup can be triggered in different ways. One option is to use `minion_task_triggers`
+and specify tasks via `on_job_done`. Another way to do that is to use the systemd timers
+`openqa-enqueue-*-cleanup` to periodically run tasks. Both can be used separately or in combination.
+
+The relevant Minion tasks are:
+
+- limit_assets
+- limit_audit_events
+- limit_bugs
+- limit_results_and_logs
+- limit_screenshots
+
+If a task is already running or nothing needs to be cleaned up these are no-ops so
```

Comment:
The asset cleanup actually is a bit more involved. One a small instance it is really just running a few seconds but e.g. on our production instance even the computation of *what* to delete can take some time. Not sure how long but definitely long enough for having to speed it up (https://github.com/os-autoinst/openQA/pull/4189).

Reply:
Afaik it will not compute anything if the lock's already being held.

Reply:
I was getting at the "or nothing needs to be cleaned up" part.